src/util/color_spaces.ts

```diff
@@ -53,9 +53,14 @@ export function rgbToLab([r, g, b, alpha]: RGBColor): LABColor {
     r = rgb2xyz(r);
     g = rgb2xyz(g);
     b = rgb2xyz(b);
-    const x = xyz2lab((0.4124564 * r + 0.3575761 * g + 0.1804375 * b) / Xn);
-    const y = xyz2lab((0.2126729 * r + 0.7151522 * g + 0.0721750 * b) / Yn);
-    const z = xyz2lab((0.0193339 * r + 0.1191920 * g + 0.9503041 * b) / Zn);
+    let y, x, z;
+    y = xyz2lab((0.2225045 * r + 0.7168786 * g + 0.0606169 * b) / Yn), x, z;
```

Comment:
What does "x, z;" do here?

Reply:
Oops -- nothing, I think. Left over from a different way of having the variables defined earlier.